crates/apub/src/activities/receive/comment.rs

```diff
@@ -1,4 +1,4 @@
-use crate::{activities::receive::get_actor_as_user, objects::FromApub, ActorType, NoteExt};
+use crate::{activities::receive::get_actor_as_person, objects::FromApub, ActorType, NoteExt};
```

Comment:
Probably good for you to look over every change in the apub folder, that's your area of expertise.